ops/model.py

```diff
@@ -2931,16 +2931,20 @@ def get_pebble(self, socket_path: str) -> 'Client':
     def planned_units(self) -> int:
         """Count of "planned" units that will run this application.
 
-        Includes the current unit in the count.
+        Includes the current unit in the count, but does not include dying units.
```

Comment:
This will include the current unit, any units that are alive, units that are in the process of being started, but will not include units that are being shut down.